pipeline/alert.go

```diff
@@ -963,6 +968,74 @@ type SensuHandler struct {
 	*AlertNode
 }
 
+// Send the alert to Pushover.
+// Register your application with Pushover at
+// https://pushover.net/apps/build to get a
+// Pushover token.
+//
+// Alert Level Mapping:
+// OK - Sends a -2 priority level.
+// Info - Sends a -1 priority level.
+// Warning - Sends a 0 priority level.
+// Critical - Sends a 1 priority level.
+//
+// Example:
+//    [pushover]
+//      enabled = true
+//      token = "9hiWoDOZ9IbmHsOTeST123ABciWTIqXQVFDo63h9"
+//      user_key = "Pushover"
+//
+// Example:
+//    stream
+//         |alert()
+//             .pushover()
+//              .sound('siren')
+//              .user_key('other user')
+//              .device('mydev')
+//              .title('mytitle')
+//              .URL('myurl')
+//              .URLTitle('mytitle')
+//
+// Send alerts to Pushover.
+//
+// tick:property
+func (a *AlertNode) Pushover() *PushoverHandler {
+	pushover := &PushoverHandler{
+		AlertNode: a,
+	}
+	a.PushoverHandlers = append(a.PushoverHandlers, pushover)
+	return pushover
+}
+
+// tick:embedded:AlertNode.Pushover
+type PushoverHandler struct {
+	*AlertNode
+
+	// User/Group key of your user (or you), viewable when logged
+	// into the Pushover dashboard. Often referred to as USER_KEY
+	// in the Pushover documentation.
+	// If empty uses the user from the configuration.
+	UserKey string
```

Comment:
Ah, this is probably due to my own confusion. I was under the impression that UserKey wasn't sensitive. From the [Pushover documentation](https://pushover.net/api#identifiers)

> Once you have an API token for your application, you'll need the user key and optional device name for each user to which you are pushing notifications.

But thinking about it a bit more makes me think that the user token should be secret.